### PR TITLE
feat: button recipe creation, loader is ok

### PR DESCRIPTION
**Pull request by bayram-69**

### CODE REVIEW

File: frontend/src/main.jsx [hunk 1]
```diff
@@ -3,14 +3,29 @@ import ReactDOM from "react-dom/client";
...
 const router = createBrowserRouter([
   {
     path: "/",
     element: <App />,
     children: [
+      {
+        path: "/",
+        element: <HomePage />,
+        loader: async () => {
+          const response = await fetch("http://localhost:3310/api/recipe");
```
Comment (ioayoub) [inline frontend/src/main.jsx:20]: prefer using : await fetch("myurl.com").then(res => res.json)

File: frontend/src/pages/NewRecipePage.jsx [hunk 2]
```diff
@@ -0,0 +1,5 @@
+function NewRecipePage() {
```
Comment (ioayoub) [inline frontend/src/pages/NewRecipePage.jsx:1]: ❌ - Empty files are not allowed

**Review by ioayoub — CHANGES_REQUESTED** [review]

**Review by QuentinVerwaerde — APPROVED** [review]

**Review by PierreA44 — APPROVED** [review]

**Review by Mewlyscin — APPROVED** [review]

File: backend/seed.js [hunk 3]
```diff
@@ -20,14 +20,21 @@ const seed = async () => {
     // Generating Seed Data
 
     // Optional: Truncate tables (remove existing data)
-    await database.query("truncate item");
+    await database.query("DELETE FROM recipe");
```
Comment (ioayoub) [inline backend/seed.js:23]: Prefer a TRUNCATE here

File: frontend/src/main.jsx [hunk 4]
```diff
@@ -2,14 +2,26 @@ import React from "react";
...
 const router = createBrowserRouter([
   {
     path: "/",
     element: <App />,
     children: [
+      {
+        path: "/",
+        element: <HomePage />,
+        loader: async () => {
+          const recipes = await fetch("http://localhost:3310/api/recipe").then(
```
Comment (ioayoub) [inline frontend/src/main.jsx:19]: Use env variable instead of complete url

**Review by ioayoub — CHANGES_REQUESTED** [review]

**Review by akornne — APPROVED** [review]

File: frontend/src/components/Recipes.jsx [hunk 5]
```diff
@@ -0,0 +1,5 @@
+function Recipes() {
```
Comment (ioayoub) [inline frontend/src/components/Recipes.jsx:1]: Empty files are not allowed

**Review by ioayoub — CHANGES_REQUESTED** [review]

**Review by ioayoub — APPROVED** [review]

**Pull request merged**